ncm-metaconfig/src/main/metaconfig/beats/pan/schema_7.0.pan

```diff
@@ -163,24 +171,96 @@ type beats_service = {
     'topology_expire' ? long(0..)
     'geoip' ? beats_shipper_geoip
     'seccomp.enabled' ? boolean
+    'seccomp' ? beats_seccomp
 };
 
 @documentation{
     Handle logmessages spread over multiple lines
 }
 type beats_filebeat_input_multiline = {
-    'pattern' ? string_trimmed
+    'pattern' ? string
```

Comment:
If this field really needs allow leading/trailing white-space, then it might be good to add an annotation to reflect this, so that someone doesn't try to "fix" it again in future.